@xen-orchestra/web/src/composables/xo-backup-job-schedules.composable.ts

```diff
@@ -0,0 +1,47 @@
+import type { XoBackupJob } from '@/remote-resources/use-xo-backup-job-collection.ts'
+import { useXoBackupLogCollection } from '@/remote-resources/use-xo-backup-log-collection'
+import { useXoScheduleCollection } from '@/remote-resources/use-xo-schedule-collection'
+import type { XoBackupLog } from '@/types/xo/backup-log.type'
+import type { IconName } from '@core/icons'
+import { createMapper } from '@core/packages/mapper'
+import { useI18n } from 'vue-i18n'
+
+export function useXoBackupJobSchedules() {
+  const { t } = useI18n()
+
+  const { getLastNBackupLogsByJobId } = useXoBackupLogCollection()
+  const { schedulesByJobId } = useXoScheduleCollection()
+
+  const getRunStatusIcon = createMapper<XoBackupLog['status'], IconName>(
+    {
+      success: 'legacy:status:success',
+      skipped: 'legacy:status:warning',
+      interrupted: 'legacy:status:danger',
+      failure: 'legacy:status:danger',
+      pending: 'legacy:status:info',
+    },
+    'failure'
+  )
+
+  const getRunInfo = (backupLog: XoBackupLog, index: number) => ({
+    icon: getRunStatusIcon(backupLog.status),
+    tooltip: `${t('last-run-number', { n: index + 1 })}: ${new Date(backupLog.end ?? backupLog.start).toLocaleString()}, ${t(backupLog.status)}`,
```

Comment:
Here you can apply the fix made by @J0ris-K to add reactivity and format the date correctly:
```suggestion
    tooltip: `${t('last-run-number', { n: index + 1 })}: ${d(backupLog.end ?? backupLog.start, 'datetime_short')}, ${t(backupLog.status)}`,
```